docs/application/native/guides/app-management/efl-ui-app.md

```diff
@@ -261,4 +262,4 @@ An application with a background running capability must declare the background
 ```
 The `<metadata key="http://tizen.org/metadata/bacgkround-category/<value>"/>` element has no effect on Tizen 2.3 devices, but on Tizen 2.4 and higher devices, it has the same effect as the `<background-category>` element.
 
-The background category of your application can be specified in the [application project settings](../../../../org.tizen.training/html/native/process/setting_properties_n.htm#manifest) in the Tizen Studio.
\ No newline at end of file
+The background category of your application can be specified in the [application project settings](../../../../org.tizen.training/html/native/process/setting_properties_n.htm#manifest) in the Tizen Studio.
```

Comment:
rf) `../../../tutorials/process/setting-properties.md` would be correct

Reply:
The file name is still ../../tutorials/process/setting-properties-n.md.
Removing '-n' from filename under native directory is done by other members.

Moreover, I'm checking wrong links and modinging TOC.md on https://github.com/journey2w/tizen-docs/tree/dotnet-toc/docs.
I will double check this link error after removing '-n' from filename.